.pre-commit-config.yaml

```diff
@@ -0,0 +1,8 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.2.0
+    hooks:
+    -   id: trailing-whitespace
+    -   id: end-of-file-fixer
```

Comment:
Prettier handles both of these, so these can be removed in favor of the Prettier pre-commit hook.
```suggestion
```

Reply:
@CKCarr I'm good with this with @helmturner 's requested changes. Mark it ready for review when those are implemented and either one of us can approve it and get it merged 🎉 :shipit: